sdk/storage/azure-storage-blob/tests/test_blob_access_conditions.py

```diff
@@ -621,6 +621,7 @@ def test_set_blob_properties_with_if_unmodified_fail(self, resource_group, locat
         # Assert
         self.assertEqual(StorageErrorCode.condition_not_met, e.exception.error_code)
 
+    @pytest.mark.playback_test_only
```

Comment:
why playback only?

Reply:
somehow the tenant doesn't have this feature enabled yet